tests/tests_fabric/test_cli.py

```diff
@@ -65,6 +67,15 @@ def test_cli_env_vars_strategy(_, strategy, monkeypatch, fake_script):
     assert os.environ["LT_STRATEGY"] == strategy
 
```

Comment:
Can we also add a test that just tests the values of `_get_supported_strategies`? Either by length or by comparing the actual values?

Reply:
Is this the same request as your suggestion above. If not, I don't understand what you meant.

Reply:
the request above is to test this for every strategy. Here I want something like

```
assert len(_get_supported_strategies()) == 42
assert "deepspeed" in _get_supported_strategies()
...
```

Reply:
Got it. added